esi_leap/tests/api/controllers/v1/test_offer.py

```diff
@@ -105,6 +115,41 @@ def test_post(self, mock_aoa, mock_create, mock_cra, mock_generate_uuid):
         self.assertEqual(data, request.json)
         self.assertEqual(http_client.CREATED, request.status_int)
 
+    @mock.patch('oslo_utils.uuidutils.generate_uuid')
+    @mock.patch('esi_leap.api.controllers.v1.utils.check_resource_admin')
+    @mock.patch('esi_leap.objects.offer.Offer.create')
+    @mock.patch('esi_leap.api.controllers.v1.offer.' +
+                'OffersController._add_offer_availabilities')
+    def test_post_default_resource_type(self, mock_aoa, mock_create, mock_cra,
+                                        mock_generate_uuid):
+        mock_generate_uuid.return_value = self.test_offer.uuid
+        mock_aoa.return_value = self.test_offer_default_resource_type.to_dict()
+
+        data = {
+            "resource_uuid": self.test_offer.resource_uuid,
+            "name": self.test_offer.name,
+            "start_time": "2016-07-16T00:00:00",
+            "end_time": "2016-10-24T00:00:00"
+        }
+
+        request = self.post_json('/offers', data)
+
+        data['project_id'] = self.context.project_id
+        data['uuid'] = self.test_offer_default_resource_type.uuid
+        data['status'] = statuses.AVAILABLE
+        data['resource_type'] = 'ironic_node'
+        data['resource_uuid'] = \
+            self.test_offer_default_resource_type.resource_uuid
```

Comment:
data['resource_uuid'] before sending the post request is self.test_offer.resource_uuid, and now it's set as self.test_offer_default_resource_type.uuid. I don't get it, though it won't effect the test.

Reply:
Good point! I've updated the test to be more consistent